File: classes/Administrator.java
public class Administrator {
    // Fields
    private String adminName;
    private String email;

    // Constructor
    public Administrator(String adminName, String email) {
        this.adminName = adminName;
        this.email = email;
    }

    // Getter & Setter Methods
    public String getAdminName() {
        return adminName;
    }

    public void setAdminName(String adminName) {
        this.adminName = adminName;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: javascript classes/administrator.js
class Administrator {
    // Constructor
    constructor(adminName, email) {
        this.adminName = adminName;
        this.email = email;
    }

    // Getters & Setters
    getAdminName() {
        return this.adminName;
    }

    setAdminName(adminName) {
        this.adminName = adminName;
    }

    getEmail() {
        return this.email;
    }

    setEmail(email) {
        this.email = email;
    }
}